.github/workflows/ci.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       - name: Check docstrings
         run: pydocstyle gnomad
       - name: Run Pylint
-        run: ./lint --disable=W --enable=logging-not-lazy,logging-format-interpolation,logging-fstring-interpolation
+        run: ./lint
```

Comment:
Thank you for fixing!

Reply:
This was working fine. Changing from disable all + re-enable some to disable some just clarifies the list of what's remaining to be resolved.